### ISSUE TITLE
Feature request: showing moved lines differently

### ISSUE THREAD

**Issue by gnutix** [banga/git-split-diffs#28]
One thing I ([and others](https://github.com/isaacs/github/issues/729)) find sad about GitHub's diff is that it doesn't show succintly moved lines (it shows one line removed, one line added...). I was wondering if this project could improve that when using git diff locally ? WDYT ?

**Comment by banga** [comment]
Yes this is something Phabricator does much better. I think modifying the diff algorithm itself requires more information than we have in this tool (since it's just parsing the output of git diff), so it'll be hard to incorporate something like that here. I know there is `git diff --color-moved`, but it doesn't format the diff differently for moved lines, so that information gets lost.

**Comment by chtenb** [comment]
Could we perhaps interpret the color bytes somehow?
